src/test/java/uk/gov/digital/ho/hocs/queue/common/MessageTypes.java

```diff
@@ -2,7 +2,8 @@
 
 public enum MessageTypes {
     UKVI_COMPLAINTS("UKVI_COMPLAINTS"),
-    UNKNOWN("UNKNOWN");
+    UNKNOWN("UNKNOWN"),
+    MIGRATED_CASES("MIGRATED_CASES");
```

Comment:
Same for this if we change this to `MIGRATION`.

Reply:
Changed